docs/get-started/getting-started-gcp.md

```diff
@@ -129,7 +129,7 @@ submitted as a Dataproc job.  The mortgage examples we use above are also availa
 application](https://github.com/NVIDIA/spark-xgboost-examples/tree/spark-3/examples/apps/scala).
 After [building the jar
 files](https://github.com/NVIDIA/spark-xgboost-examples/blob/spark-3/getting-started-guides/building-sample-apps/scala.md)
-they are available through maven `mvn package -Dcuda.classifier=cuda10-2`.
+they are available through maven `mvn package`.
```

Comment:
No need change here. Just reminder us to change the spark-xgboost-examples repo.(remove the classifier)